src/utils/client.ts

```diff
@@ -1,11 +1,14 @@
 // Replace the package name with your application's package name instead.
-import { FoundryBrowserClient } from "@ontology-starter/sdk";
+import { FoundryClient, PublicClientAuth } from "@ontology-starter/sdk";
 
 /**
  * Initialize the client to interact with the Ontology SDK
  */
-export const client = new FoundryBrowserClient({
-    clientId: process.env.FOUNDRY_CLIENT_ID!,
+export const client = new FoundryClient({
     url: process.env.FOUNDRY_API_URL!,
-    redirectUrl: process.env.APPLICATION_REDIRECT_URL!,
+    auth: new PublicClientAuth({
+        clientId: process.env.FOUNDRY_CLIENT_ID!,
+        url: process.env.FOUNDRY_API_URL!,
```

Comment:
hm.. now that this can be separate, i might explore seeing if it's necessary to have https anymore